extensions/ql-vscode/syntaxes/ql.tmLanguage.yml

```diff
@@ -113,7 +119,7 @@ repository:
     name: keyword.operator.relational.ql
   
   comparison-operator:
-    match: '=|\!-'
+    match: '=|\!\='
```

Comment:
Was this a bug?

Reply:
Yes, that's a trivial bug I noticed while I was in the code. With the default color settings, it doesn't affect anything, because operators don't get a special color. In custom color schemes, though, we would highlight `!-` instead of `!=`.

Reply:
Ah. Makes sense. Good catch.